user/for-beginners.md

```diff
@@ -1,57 +1,53 @@
 ---
-title: Travis CI for Complete Beginners
+title: Travis CI for Beginners
 layout: en
 permalink: /user/for-beginners/
 ---
 
-If you're not familiar with continuous integration and don't already have a repository that you want to build, this guide will show you what it is all about.
+Welcome to Travis CI! If you are new here or new to Continuous Integration (CI) in general, this doc will provide context and terminology used throughout the platform and documentation. 
 
-You need to sign up for a [GitHub account](https://github.com/) if you do not already have one.
+### What is Continuous Integration (CI)?
 
-### To get started with Travis CI:
+Continuous Integration is the concept and practice of merging in small code changes frequently - rather than waiting for a large integration process at the end of a development cycle. Companies and individuals may all do this process differently, but the goal is to build healthier software by developing and testing in smaller increments. This is where Travis CI can be helpful.
 
-1. On GitHub, fork the [example PHP repository](https://github.com/plaindocs/travis-broken-example).
+Travis CI, as a CI platform, supports your development process by automatically building and testing code changes, in order to provide immediate feedback on the success of the change. Travis CI also can automate some of your development process by managing deployments and notifications.  
```

Comment:
@plaindocs I can't remember if we discussed this... but I think we should include reference to it. That said, I'll tighten it up a bit. Plus, the way it reads sounds like we have a separate artifacts management feature which, um, is not the case at the moment.